### DISCUSSION TITLE
關於CPE的環境「瘋狂程設」

### DISCUSSION THREAD

**Discussion by davidteng00000** [1101-GS4719-C/Python-Community-GS4719-C#1664] [General]
在瘋狂程設環境中，雖然操作介面不友善，但是只要更改編譯器到"minGW"後，便可以使用C++的萬用標頭檔<bits/stdc++.h>了! 而且他給的範例測資還不少，比Online Judge完全不透明的方式好一些。缺點就是不能複製已經打好的程式碼到其他題目。而且10/17CPE的power 函式竟然是壞掉的! 害我們的競程大神debug100分鐘才過第二題，太糟糕了!

希望各位都可以對超過2題!

**Comment by v0ouo** [comment]
我一開始一直CE，後來才發現是編譯器問題，然後那個環境也超爛，我第一次用到打左括號不會幫你補右括號的，比DEVC++還爛，除了測資給了多一點，基本上沒什麼優點，跑得還超慢，總之考一次有過畢業門檻就好，應該不會再考下一次了。

**Comment by Cerosop** [comment]
我前幾次用power涵式他也是壞掉的，只能自己寫一個power出來，這次我甚至後面for迴圈也壞了，超級爛。

**Comment by mrweixd** [comment]
我在cin測資的時候也可以出錯，超級不可思議。我後來刪掉全部重寫才恢復正常，浪費掉超多時間ＱＱ。謝謝你瘋狂程設，原來是讓我們瘋掉的程式設計嗎
